stack-graphs/src/c.rs

```diff
@@ -1989,16 +1989,23 @@ pub extern "C" fn sg_forward_partial_path_stitcher_from_nodes(
 /// `previous_phase_partial_paths_length` fields.
 #[no_mangle]
 pub extern "C" fn sg_forward_partial_path_stitcher_from_partial_paths(
-    _graph: *const sg_stack_graph,
+    graph: *const sg_stack_graph,
     partials: *mut sg_partial_path_arena,
-    _db: *mut sg_partial_path_database,
+    db: *mut sg_partial_path_database,
```

Comment:
Woohoo, I knew having consistent function signatures would pay off!

Reply:
I went to this same thought process :) First I left them out when they weren't used. But a path can only be interpreted with the corresponding graph/arena/db. The fact that some things are available without is only an implementation detail.